README.md

```diff
@@ -1,2 +1,4 @@
-# Татарская Android клавиатура
-## Tatar language Android keyboard
+# Татарская Android AnySoftKeyboard клавиатура
+## Tatar language pack for AnySoftKeyboard Android keyboard
+To configure signing, please, create `keystore.properties` file with content for signing like `storeFile`, `storePassword`, `keyAlias`, `keyPassword`.
+The dictionary was compiled from old version of `makedict` and `generatewordlist` (find them here :https://code.google.com/archive/p/softkeyboard/source/default/source) 
```

Comment:
Regarding your dictionary, there is a `makeDictionary`gradle task to generate a binary dict file out of variety of sources. Try that.